uconfig.h

```diff
@@ -3594,7 +3594,7 @@
 
 /* HAS_WCSXFRM:
  *	This symbol, if defined, indicates that the wcsxfrm routine is
- *	available to tranform a wide character string for wcscmp().
+ *	available to transform a wide character string for wcscmp().
```

Comment:
This change should be made in `config_h.SH`.  This change will be overridden on a make regen.

Reply:
updated with other generated files

Reply:
I updated this in `metaconfig/U/perl/d_wcsxfrm.U` as 87e76cbf7e9ee95ecd1bfe81e6ddd31351ada25f
will be included after the next regen